aibolit/patterns/er_class/er_class.py

```diff
@@ -1,14 +1,15 @@
-import javalang
-
+from aibolit.ast_framework import ASTNodeType, AST
 from aibolit.utils.ast_builder import build_ast
+from typing import List
 
 
 class ErClass:
 
     def __init__(self):
```

Comment:
I think we can remove empty constructors.

Reply:
done